src/content/docs/tutorial-errors/analyze-your-errors.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Analyze error details
+metaDescription: Learn how to manage a high number of errors in your app.   
+---
+
+import apmErrorsAttributes from 'images/apm_screenshot-crop_errors-details.webp'
+
+import apmLogs from 'images/apm_screenshot-crop_errors-logs.webp'
+
+import apmLogsZoom from 'images/apm_screenshot-crop_errors-logs-zoomed.webp'
+
+import apmStackTraces from 'images/apm_screenshot-crop_errors-stack-traces.webp'
+
+import apmTriageError from 'images/apm_screenshot-crop_errors-triage.webp'
+
+Once you've chosen your error group, New Relic takes you to a new page that displays attributes, logs, and traces about the error. This part of the tutorial guides you through two recommended avenues for analyzing an error, your logs and stack traces. 
+
+## Objectives [#objectives]
+
+This doc builds off the problem scoped from the [Scope the problem](/docs/tutorial-errors/scope-the-problem) doc of this tutorial series. In this doc, you will:
+
+* Find the error code associated with a failing API with logs
+* Use stack traces to identify faulty code logic or mistype code  
+
+## Find the source of an error with logs and stack traces [#analyze]
```

Comment:
I'd emphasize stack traces and maybe bring them to the forefront stylistically.  I hear about them a lot and I think they're a very critical part of the product. Have you spoken to Cheryl Frankenfield? She might be a good SME to speak to about this.